zubhub_backend/zubhub/activitylog/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path
+from .views import *
+
+app_name = "activitylog"
+
+urlpatterns = [
+    path('',AllUsersActivitylogAPIView.as_view(), name='all-activitylogs' ),
+    path('<str:username>/',
+         UserActivitylogAPIView.as_view(), name='user-activitylog')
+    # path('<str:username>',
```

Comment:
Remove these lines or add a comment if they are here for a reason.

Reply:
Done.